tests/test_all.py

```diff
@@ -58,38 +58,47 @@ def test_os_release(auto_container):
     """
     assert auto_container.connection.file("/etc/os-release").exists
 
-    for var_name, value in (
+    for var_name, expected_value in (
         ("VERSION_ID", OS_VERSION),
         ("PRETTY_NAME", OS_PRETTY_NAME),
     ):
-        if OS_VERSION == "tumbleweed" and var_name == "VERSION_ID":
-            # on openSUSE Tumbleweed that is the an ever changing snapshot date
-            # just check whether it starts with current year
-            assert (
-                int(
-                    auto_container.connection.run_expect(
-                        [0], f". /etc/os-release && echo ${var_name}"
-                    ).stdout.strip()[:4]
+        if var_name == "VERSION_ID":
+            if OS_VERSION == "basalt":
+                # Current basalt version number
+                expected_value = "0.1"
```

Comment:
Wouldn't this be better suites as a constant in `bci_tester/data.py`? :thinking:

Reply:
I have moved it to data.py